content/200-concepts/100-components/02-prisma-client/260-client-extensions/index.mdx

```diff
@@ -63,26 +63,47 @@ generator client {
 
 ## Create an extension
 
-Use the `$extends` [client-level method](/reference/api-reference/prisma-client-reference#client-methods) to create an extended client. In your `$extends` structure, use the appropriate extension component or components ([`model`](/concepts/components/prisma-client/client-extensions/model), [`client`](/concepts/components/prisma-client/client-extensions/client), [`result`](/concepts/components/prisma-client/client-extensions/result) or [`query`](/concepts/components/prisma-client/client-extensions/query)).
+You can create an extension using two primary ways. You can either:
 
-For example, the following code uses a [`model` extension component](/concepts/components/prisma-client/client-extensions/model) to extend the `User` model.
+1. Use `$extends` [client-level method](/reference/api-reference/prisma-client-reference#client-methods)
 
-```ts
-const xprisma = prisma.$extends({
-  name: `signUp`, // Optional: name appears in error logs
-  model: {        // This is a `model` component
-    user: { ... } // The extension logic for the `user` model goes inside the curly braces
-  },
-})
-```
+   ```ts
+   const xprisma = prisma.$extends({
+    name: 'signUp', // Optional: name appears in error logs
+    model: {        // This is a `model` component
+      user: { ... } // The extension logic for the `user` model goes inside the curly braces
+    },
+   })
+   ```
```

Comment:
```suggestion
  ```ts
  const xprisma = prisma.$extends({
    name: 'signUp', // Optional: name appears in error logs
    model: {        // This is a `model` component
      user: { ... } // The extension logic for the `user` model goes inside the curly braces
    },
  })
  ```
```